### DISCUSSION TITLE
will scores with rate changes be ranked after rates get ranked if they were set before this rework?

### DISCUSSION THREAD

**Discussion by ArtemChik103** [ppy/osu#30362] [Q&A]
i've set many scores with rate changes through lazer in mania and wanna know

**Comment by mixalisnew** [comment]
Yes i am quite confident since they will just have to submit them or recalculate them and the number of rate adjusted scores are not so many.
Devs can confirm if i am wrong.

**Reply by ArtemChik103** [reply]
> Yes i am quite confident since they will just have to submit them or recalculate them and the number of rate adjusted scores are not so many. Devs can confirm if i am wrong.

yeah i hope so too

**Comment by Tenexxt** [comment]
As said in #29432 this will depend on how hard locating and ranking all scores with custom rates will be, but the initial goal is to make it happen
